azure-servicebus/src/main/java/com/microsoft/azure/servicebus/BrowsableMessageSession.java

```diff
@@ -129,7 +129,7 @@ public ReceiveMode getReceiveMode() {
     }
 
     @Override
-    public CompletableFuture<Collection<Instant>> renewMessageLockBatchAsync(Collection<? extends IMessage> messages) {
+    public CompletableFuture<Collection<Instant>> renewMessageLockBatchAsync(UUID[] lockTokens) {
```

Comment:
>CompletableFuture [](start = 11, length = 17)

Must implement renewMessageLockAsync(UUID lockToken) and throw excpetion. Also implement renewMessageLockBatchAysnc(Collection<? extends IMessage).

Reply:
renewMessageLockBatchAysnc(Collection<? extends IMessage) has been changed to this function. 

---
In reply to: [138766129](https://github.com/Azure/azure-service-bus-java/pull/127#discussion_r138766129) [](ancestors = 138766129)